json_formatter.go

```diff
@@ -43,6 +54,21 @@ type JSONFormatter struct {
 	// }
 	FieldMap FieldMap
 
+	// LogLevelMap allows users to customize the values used for the various log levels
+	// As an example:
+	// formatter := &JSONFormatter{
+	//   	LogLevelMap: LogLevelMap{
+	// 		 PanicLevel:  1,
```

Comment:
This example isn't valid. The value on the right side needs to be a string, so "1", "2", "3", ...